modules/default/weather/weatherobject.js

```diff
@@ -10,6 +12,13 @@
  * As soon as we start implementing the forecast, mode properties will be added.
  */
 class WeatherObject {
+	/**
+	 *
+	 * @param units
```

Comment:
what about all the following warnings `Missing JSDoc @param ...`?

Reply:
yeah, will fix those with the next commits over the weekend. stay tuned :-)